bin/deploy

```diff
@@ -74,6 +75,28 @@ test -f "$LSSTSW/lfs/.git.deployed" || ( # git
     touch "$LSSTSW/lfs/.git.deployed"
 )
 
+test -f "${LSSTSW}/lfs/.git-lfs.deployed" || (
```

Comment:
The cleanups were more noise then is reasonable in a single PR, and travis seems to be having  a problem installing shellcheck, so I've split it off as #53 which can be rebased after this PR is merged..